src/Umbraco.Web.UI.Client/src/views/packages/options.controller.js

```diff
@@ -21,13 +21,6 @@
 
                 vm.loading = false;
 
-                //make sure the packageView is formatted as a virtual path
```

Comment:
This should not be moved to the `convertVirtualToAbsolutePath` method, as that method should not be handling relative paths. It might work for this, but if used elsewhere that method would need to throw exceptions and not just "handle" a relative path.